### ISSUE TITLE
Govspeak CTA colour contrast issue in `whitehall` frontend

### ISSUE THREAD

**Issue by danacotoran** [alphagov/govuk_publishing_components#2706]
Not sure if this is already known, haven't seen anything about in the list of issues on this repo but please feel free to close it if it's already in someone's backlog.

On the whitehall page in the screenshot below, the `govspeak` "call to action" element background colour is not appropriate for the colour of text and a WCAG fail. 

Curiously, it looks just fine in the component guide: https://components.publishing.service.gov.uk/component-guide/govspeak/call_to_action 
So I wonder if this is a legacy css issue affecting only `whitehall-frontend`.

Can we investigate whether the dark grey colour for `.call-to-action` can be changed to a lighter grey so it doesn't fail minimum colour contrast requirements against our standard text and link colours?

-----

![Screenshot 2022-03-30 at 17 15 36](https://user-images.githubusercontent.com/7116819/160883424-155cf7e3-c8ee-43fb-ab7f-66040197b3e1.png)

-----

The page in question that the screenshot is from: https://www.gov.uk/government/topical-events/russian-invasion-of-ukraine-uk-government-response

**Comment by andysellick** [comment]
Example of the call to action in the govspeak component: https://components.publishing.service.gov.uk/component-guide/govspeak/image_fractions

**Comment by MartinJJones** [comment]
Closing this issue as fixed in https://github.com/alphagov/whitehall/pull/6572

**Comment by MartinJJones** [comment]
Closing this issue as fixed in https://github.com/alphagov/whitehall/pull/6572
